lib/ClangImporter/ClangImporter.cpp

```diff
@@ -2889,3 +2889,14 @@ void ClangImporter::Implementation::dumpSwiftLookupTables() {
   llvm::errs() << "<<Bridging header lookup table>>\n";
   BridgingHeaderLookupTable->dump();
 }
+
+DeclName ClangImporter::
+analyzeImportedName(const clang::NamedDecl *ClangDecl,
```

Comment:
Can you change the verb here? "analyze" doesn't really mean anything.

It really seems like this is just a stripped-down version of ClangImporter::Implementation::importFullName, so maybe just call it importName and forward to that method?

(Under what conditions would the `version` parameter be different from the AST context's language options? That seems a little iffy too.)